spec/record_level/validate_marc_spec.rb

```diff
@@ -56,13 +56,19 @@
       [
         { '011' => { 'indicator1' => ' ',
                      'indicator2' => ' ',
-                     'subfields' => [{ 'a' => 'Invalid field' }] } }
+                     'subfields' => [{ 'a' => 'Invalid field' }] } },
+        { 'ABC' => { 'indicator1' => ' ',
+                     'indicator2' => ' ',
+                     'subfields' => [{ 'a' => 'Non-numeric tag' }] } }
       ]
     end
     let(:record) { MARC::Record.new_from_hash('fields' => fields) }
     it 'finds the invalid tag' do
```

Comment:
Not sure what this block is doing. The code only checks for non-numerical characters in the field tag. This block isn't really necessary.